cmd/statusd-cli/repl.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// REPL implements the read-eval-print loop for the commands
+// to be sent to statusd.
+type REPL struct {
+	conn   net.Conn
+	reader *bufio.Reader
+	writer *bufio.Writer
+}
+
+// NewREPL creates a REPL instance communicating with the
+// addressed statusd.
+func NewREPL(serverAddress string) (*REPL, error) {
+	conn, err := net.Dial("tcp", serverAddress+":51515")
```

Comment:
Why hardcoding a port number?

Reply:
As mentioned above, simplicity for a pure debugging interface. But will now changed back to parameter again like in first draft.